packages/website/lib/subscribe.js

```diff
@@ -0,0 +1,29 @@
+import constants from './constants'
+
+const API = constants.API
+
+/**
+ *
+ * @param {string} email
+ * @param {string} [version]
+ */
+
+export const subscribe = async (email, version = '') => {
+  const loginURL = version
+    ? `/v${version}/internal/blogSubscribe`
+    : '/internal/blogSubscribe'
+  const res = await fetch(API + loginURL, {
+    method: 'POST',
+    body: JSON.stringify({
+      email,
+    }),
+  })
+  const body = await res.json()
+  if (body.ok) {
+    return body.value
+  } else {
+    if (JSON.parse(body.response.text).title === 'Member Exists')
+      throw new Error('exists')
+    throw new Error(body.error)
+  }
```

Comment:
I think the member exists thing does not happen - they get updated, and this always succeeds unless network error or something.

```suggestion
  if (!body.ok) {
    throw new Error(body.error)
  }
```

Reply:
sorry this was prior to api refactor